src/snowflake/snowpark/lineage.py

```diff
@@ -149,6 +151,7 @@ class _DGQLQueryBuilder:
     USER_TO_SYSTEM_DOMAIN_MAP = {
         _UserDomain.FEATURE_VIEW: _SnowflakeDomain.TABLE,
         _UserDomain.MODEL: _SnowflakeDomain.MODULE,
+        _UserDomain.SERVICE: _SnowflakeDomain.SNOWSERVICE_INSTANCE,
```

Comment:
are we handling the input case sensitivity?

Reply:
yep. We convert the input to upper case and then check for mapping.